src/BenchmarkDotNet/Parameters/ParameterInstance.cs

```diff
@@ -55,6 +55,10 @@ public string ToDisplayText(CultureInfo cultureInfo)
             }
         }
 
+        public string ToDisplayText(SummaryStyle summary) => ToDisplayText(summary.CultureInfo, summary.MaxParameterColumnWidth);
```

Comment:
From the CI logs, it looks that `summary` can be `null` here? Could you please fix it (by checking if we can avoid passing `null` here or making sure the method works fine when a `null` is passed)?

```log
Error Message:
 System.NullReferenceException : Object reference not set to an instance of an object.
Stack Trace:
   at BenchmarkDotNet.Parameters.ParameterInstance.ToDisplayText(SummaryStyle summary) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Parameters\ParameterInstance.cs:line 58
   at BenchmarkDotNet.Columns.ParamColumn.GetValue(Summary summary, BenchmarkCase benchmarkCase) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Columns\ParamColumn.cs:line 20
   at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at BenchmarkDotNet.Reports.SummaryTable..ctor(Summary summary, SummaryStyle style) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Reports\SummaryTable.cs:line 60
   at BenchmarkDotNet.Reports.Summary..ctor(String title, ImmutableArray`1 reports, HostEnvironmentInfo hostEnvironmentInfo, String resultsDirectoryPath, String logFilePath, TimeSpan totalTime, CultureInfo cultureInfo, ImmutableArray`1 validationErrors) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Reports\Summary.cs:line 60
   at BenchmarkDotNet.Running.BenchmarkRunnerClean.Run(BenchmarkRunInfo benchmarkRunInfo, Dictionary`2 buildResults, IResolver resolver, ILogger logger, List`1 artifactsToCleanup, String resultsFolderPath, String logFilePath, StartedClock& runChronometer) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Running\BenchmarkRunnerClean.cs:line 134
   at BenchmarkDotNet.Running.BenchmarkRunnerClean.Run(BenchmarkRunInfo[] benchmarkRunInfos) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Running\BenchmarkRunnerClean.cs:line 76
   at BenchmarkDotNet.Running.BenchmarkRunner.RunWithDirtyAssemblyResolveHelper(Type type, IConfig config) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Running\BenchmarkRunnerDirty.cs:line 77
   at BenchmarkDotNet.Running.BenchmarkRunner.RunWithExceptionHandling(Func`1 run) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Running\BenchmarkRunnerDirty.cs:line 107
   at BenchmarkDotNet.Running.BenchmarkRunner.Run(Type type, IConfig config) in C:\projects\benchmarkdotnet\src\BenchmarkDotNet\Running\BenchmarkRunnerDirty.cs:line 30
   at BenchmarkDotNet.IntegrationTests.BenchmarkTestExecutor.CanExecute(Type type, IConfig config, Boolean fullValidation) in C:\projects\benchmarkdotnet\tests\BenchmarkDotNet.IntegrationTests\BenchmarkTestExecutor.cs:line 63
   at BenchmarkDotNet.IntegrationTests.ParamsTestStaticPropertyTest.Test() in C:\projects\benchmarkdotnet\tests\BenchmarkDotNet.IntegrationTests\ParamsAttributeStaticTest.cs:line 18
```

Reply:
I added a null check, since we do have a valid fallback (by using `maxParameterColumnWidthFromConfig`). This way, the class acts exactly the same way as before, when the SummaryStyle is null, so it won't break anything for users that rely on a null SummaryStyle.
If usage without a summary style should be prohibited, I guess we should change all tests that don't have a Summary style (not that many), to use one and add another test with a null summary style that should throw an error to document that requirement, right?